application/settings.py

```diff
@@ -37,15 +37,15 @@
     'django.contrib.contenttypes',
     'django.contrib.staticfiles',
     'django.contrib.gis',
+    'django.contrib.auth',
```

Comment:
It seems like now Django loads it by default. Somehow I found only one reference to the similar error: http://stackoverflow.com/questions/34754907/django-1-9-and-django-contrib-auth and only thing they suggest is to add this `contrib.auth`